Makefile

```diff
@@ -39,12 +41,20 @@ INTEGRATION_TEST_FIXTURES = ./fixtures
 
 all: ks docs
 
-ks:
+Gopkg.lock: Gopkg.toml
+	dep ensure
+	touch Gopkg.lock
+	$(eval APIMACHINERY_VER := $(shell grep -B1 k8s.io/apimachinery Gopkg.lock | head -n1 | cut -d'"' -f2))
+
+ks: Gopkg.lock
 	$(GO) build -o $(KS_BIN) $(GO_FLAGS) ./cmd/ks
 
 docs:
 	$(DOC_GEN_FILE)
 
+docker-image: Gopkg.lock
```

Comment:
This should be be added to `.PHONY` at the bottom of the file? (and also `install` should be as well)

Reply:
will do